perma_web/api2/serializers.py

```diff
@@ -0,0 +1,242 @@
+from django.conf import settings
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.core.validators import URLValidator
+from requests import TooManyRedirects
+from rest_framework import serializers
+
+from utils import get_mime_type, mime_type_lookup
+from perma.models import LinkUser, Folder, CaptureJob, Capture, Link, Organization
+from perma.utils import ip_in_allowed_ip_range
+
+
+class BaseSerializer(serializers.ModelSerializer):
+    """ Base serializer from which all of our serializers inherit. """
+
+    def update(self, instance, validated_data):
+        """ When updating, requiring that our serializers provide a whitelist of fields that can be updated. """
```

Comment:
The LinkUserSerializer does not include a field in Meta "allowed_update_fields". Is that an oversight? Or would this method be better like this?
```
    def update(self, instance, validated_data):
        """Update object, failing validation if a serializer provides allowed_update_fields and disallowed fields are passed in."""
        if getattr(self, 'Meta.allowed_update_fields', False) and set(validated_data.keys()) - set(self.Meta.allowed_update_fields):
            raise serializers.ValidationError('Only updates to these fields are allowed: %s' % ', '.join(self.Meta.allowed_update_fields))
        return super(BaseSerializer, self).update(instance, validated_data)